test_resource_deletion_during_pvc_expansion[mgr] testcase is failing on ODF4.15 for IBM Power

Following Tier 4c testcases are failing on IBM Power:
```
tests/manage/pv_services/pvc_resize/test_resource_deletion_during_pvc_expansion.py::TestResourceDeletionDuringPvcExpansion::test_resource_deletion_during_pvc_expansion[mgr]
tests/manage/pv_services/pvc_resize/test_resource_deletion_during_pvc_expansion.py::TestResourceDeletionDuringPvcExpansion::test_resource_deletion_during_pvc_expansion[osd]
tests/manage/pv_services/pvc_resize/test_resource_deletion_during_pvc_expansion.py::TestResourceDeletionDuringPvcExpansion::test_resource_deletion_during_pvc_expansion[rbdplugin]
tests/manage/pv_services/pvc_resize/test_resource_deletion_during_pvc_expansion.py::TestResourceDeletionDuringPvcExpansion::test_resource_deletion_during_pvc_expansion[cephfsplugin]
tests/manage/pv_services/pvc_resize/test_resource_deletion_during_pvc_expansion.py::TestResourceDeletionDuringPvcExpansion::test_resource_deletion_during_pvc_expansion[rbdplugin_provisioner]
tests/manage/pv_services/pvc_resize/test_resource_deletion_during_pvc_expansion.py::TestResourceDeletionDuringPvcExpansion::test_resource_deletion_during_pvc_expansion[cephfsplugin_provisioner]
```
error being:
```
self = <ocs_ci.utility.utils.TimeoutSampler object at 0x7fff7f621bb0>

    def __iter__(self):
        if self.start_time is None:
            self.start_time = time.time()
        while True:
            self.last_sample_time = time.time()
            if self.timeout <= (self.last_sample_time - self.start_time):
>               raise self.timeout_exc_cls(*self.timeout_exc_args)
E               ocs_ci.ocs.exceptions.TimeoutExpiredError: Timed out after 240s running exec_cmd_on_pod("command"="df -kh")

ocs_ci/utility/utils.py:1288: TimeoutExpiredError
```

--- a/tests/functional/pv/pvc_resize/test_resource_deletion_during_pvc_expansion.py
+++ b/tests/functional/pv/pvc_resize/test_resource_deletion_during_pvc_expansion.py
@@ -165,11 +165,12 @@ class TestResourceDeletionDuringPvcExpansion(ManageTest):
                     continue
                 df_out = df_out.split()
                 new_size_mount = df_out[df_out.index(pod_obj.get_storage_path()) - 4]
-                if new_size_mount in [
-                    f"{pvc_size_expanded - 0.1}G",
-                    f"{float(pvc_size_expanded)}G",
-                    f"{pvc_size_expanded}G",
-                ]:
+                if (
+                    pvc_size_expanded - 0.7
+                    <= float(new_size_mount[:-1])
+                    <= pvc_size_expanded
+                    and new_size_mount[-1] == "G"
+                ):
                     log.info(
                         f"Verified: Expanded size of PVC {pod_obj.pvc.name} "
                         f"is reflected on pod {pod_obj.name}"

